portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,33 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> messages;
+  
+  @Override
+  public void init(){
+    messages = new ArrayList<>();
+    messages.add("Hi Prosper!");
+    messages.add("Wadup homie!");
+    messages.add("What's good!!!");
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello! I'm Prosper");
+    String json = convertToJson(messages);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJson(List messages){
```

Comment:
The only thing I might change in this code is to include the generic type argument here, i.e. change this to `List<String>` so it's more clear exactly what this function should expect.

(Your code will work as-is, but adding the generic type will make it more future-proof.)